pkg/setting/setting_test.go

```diff
@@ -433,12 +434,14 @@ func TestAlertingEnabled(t *testing.T) {
 		unifiedAlertingEnabled string
 		legacyAlertingEnabled  string
 		featureToggleSet       bool
+		isEnterprise           bool
 		verifyCfg              func(*testing.T, Cfg, *ini.File)
 	}{
 		{
 			desc:                   "when legacy alerting is enabled and unified is disabled",
 			legacyAlertingEnabled:  "true",
 			unifiedAlertingEnabled: "false",
+			isEnterprise:           rand.Int63()%2 == 0,
```

Comment:
This might be a stupid question, but why do we use `rand.Int63` here?

Reply:
to indicate that the value does not affect anything. Alternative would be a test case that covers the combination, which does not make any sense imo.

Reply:
Can we replace it by a const named `doesntMatterForTest` or something like this and document it there?

Reply:
Good idea. Done!